v2/concordium/types.proto

```diff
@@ -994,12 +994,7 @@ message RejectReason {
     plt.TokenId non_existent_token_id = 55;
     // The token-holder transaction failed.
     // Introduced in protocol version 9.
-    plt.TokenModuleRejectReason token_holder_transaction_failed = 56;
-    // The token-governance transaction failed.
-    // Introduced in protocol version 9.
-    plt.TokenModuleRejectReason token_governance_transaction_failed = 57;
-    // Account sending the transaction is not authorized for governing the token.
-    plt.TokenId unauthorized_token_governance = 58;
+    plt.TokenModuleRejectReason token_transaction_failed = 56;
```

Comment:
Also, I think it would be good to align on either `token` or `token_update` across the board. I think `token_update` describes it well and also aligns with the terminology used for contracts